src/MPDevice.cpp

```diff
@@ -1857,7 +1909,7 @@ bool MPDevice::tagPointedNodes(bool tagCredentials, bool tagData, bool repairAll
         }
     }
 
-    if (tagData)
+    if (tagData && isCred)
```

Comment:
Yes, but it will be processed with the general credentials like for the mini.
I can add the comment, but we only need to modify this part if there will seperate data nodes for webauthn too.